command/watch.go

```diff
@@ -102,17 +104,20 @@ func doWatch(ctx *cli.Context) {
 	if err != nil {
 		utils.Exit1With("first run failed", err)
 	}
-	for execTime := range time.NewTicker(watchInterval).C {
+	for range time.NewTicker(watchInterval).C {
 		output, err := evalCmdOutput()
 		if err != nil {
 			output += fmt.Sprintf("\n!== <%v> ==!", err)
 		}
 		if output != lastOutput {
 			msgData := bytes.NewBuffer([]byte{})
-			fmt.Fprintf(msgData, "command output for \"%s\" changed at %v\n\n", cmdStringNotation, execTime)
-			fmt.Fprintln(msgData, "== BEGIN OLD OUTPUT ==")
-			fmt.Fprint(msgData, lastOutput)
-			fmt.Fprintln(msgData, "== END OLD OUTPUT ==")
+			fmt.Fprintf(msgData, "command output for \"%s\" changed:\n\n", cmdStringNotation)
```

Comment:
How about adding a --quiet option which only outputs the new message without the `== ** ==` headers and this.

Reply:
I don't really like it, there are just 3 lines of extra output which is not annoying at all from my point of view, and it does not behave the same as how `--quite` behave in push subcommand, which is confusing.

Reply:
For me it is in some cases unnecessary. F.ex. the command in https://github.com/gotify/cli/pull/20#discussion_r277072635 the message should only contain the text and nothing more.

Reply:
Okay, let's call it `--short` then